internal/core/src/query/PlanProto.cpp

```diff
@@ -56,17 +56,17 @@ ProtoParser::PlanNodeFromProto(const planpb::PlanNode& plan_node_proto) {
         // currently, iterative filter does not support range search
         if (!search_info.search_params_.contains(RADIUS)) {
             if (query_info_proto.hints() != "") {
-                if (query_info_proto.hints() == ITERATIVE_FILTER) {
+                if (query_info_proto.hints() == "disable") {
+                    search_info.iterative_filter_execution = false;
```

Comment:
I can probably understand that if mv is turned on, iterative_filter_execution needs to be disabled, but it seems a bit hack here.